### PR TITLE
new Galayou Y4 Variant   T31L, SC2336, ATBM6012BX, 8MB

### PR DESCRIPTION
**Pull request by tobs1895**
Galayou Y4
Add new Cameramodel/Variant
Galayou Y4 with
HW:
	-T31L
	-SC2336
	-ATBM6012BX
       -8MB

I did all the testings I could do.
Works fine since 10 days for now.

### CODE REVIEW

**Comment by tobs1895** [comment]
This is my first Pullrequest!
Hope I did it the right way!

File: environment/galayou_Y4-t31l-atbm6012bx.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+day_night_max=15000
+day_night_min=500
+disable_eth=true
+enable_updates=true
+gpio_button=14
+gpio_default=7O 8o 9O 10o 11o 14i 16i 17o 18O 49o 52o 53o 57o 58o 59o 61o 62o 63o 64O
+gpio_ir940=11
+gpio_ircut=57 58
+gpio_led_b=9
```
Comment (gtxaspec) [inline environment/galayou_Y4-t31l-atbm6012bx.txt:9]: can you add a suffix to the gpios? O=gpio high to turn on or o=gpio low to turn on? for both LEDs please.

Comment (tobs1895) [reply environment/galayou_Y4-t31l-atbm6012bx.txt:9]: Is it needed, while it is set in the gpio_defaults?

File: environment/galayou_Y4-t31l-atbm6012bx.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+gpio_motor_h=49 61 62 63
+gpio_motor_v=52 59 64 53
+gpio_speaker=7o
+gpio_white=10
+gpio_wlan=17o
+motor_maxstep_h=4050
+motor_maxstep_v=800
+osmem=43520K@0x0
+rmem=22016K@0x2A80000
+wlanbus=usb
```
[the remark this answers is not in the document]
Comment (tobs1895) [reply environment/galayou_Y4-t31l-atbm6012bx.txt]: Done.
I think...

**Comment by gtxaspec** [comment]
merged thanks!

**Comment by devdems** [comment]
Hi @gtxaspec @tobs1895 how do i flash the fw to Galayou 4k?

**Pull request merged**